lib/profile/icon_button.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flutter/material.dart';
+ 
+class ProfileIconButton extends StatelessWidget {
+  final IconData name;
+  final String caption;
+  ProfileIconButton({Key key, @required this.name, @required this.caption}) : super(key: key);
```

Comment:
Great! I appreciate you generalizing this as I'll probably use this in the dashboard too!